galaxy/models.py

```diff
@@ -312,67 +288,210 @@ def compute_users_clubs_score(cls, user1, user2) -> int:
     ###################
 
     @classmethod
-    def rule(cls) -> None:
-        GalaxyStar.objects.all().delete()
-        # The following is a no-op thanks to cascading, but in case that changes in the future, better keep it anyway.
-        GalaxyLane.objects.all().delete()
+    def scale_distance(cls, value) -> int:
+        # TODO: this will need adjustements with the real, typical data on Taiste
+        if value == 0:
+            return 4000  # Following calculus would give us +∞, we cap it to 4000
+
+        cls.logger.debug(f"\t\t> Score: {value}")
+        # Invert score to draw close users together
+        value = 1 / value  # Cannot be 0
+        value += 2  # We use log2 just below and need to stay above 1
+        value = (  # Let's get something in the range ]0; log2(3)-1≈0.58[ that we can multiply later
+            math.log2(value) - 1
+        )
+        value *= (  # Scale that value with a magic number to accommodate to typical data
+            # Really close galaxy citizen after 5 years typically have a score of about XXX
+            # Citizen that were in the same year without being really friends typically have a score of about XXX
+            # Citizen that have met once or twice only have a couple of pictures together typically score about XXX
+            cls.GALAXY_SCALE_FACTOR
+        )
+        cls.logger.debug(f"\t\t> Scaled distance: {value}")
+        return int(value)
+
+    def rule(self, picture_count_threshold=10) -> None:
```

Comment:
What is `picture_count_threshold` ?
Could add some documentation to this function ?

Reply:
Done